### PR TITLE
ci: create fledge.yaml

### PR DESCRIPTION
**Pull request by maelle**
@krlmlr btw fledge is super behind on version bumping :sweat_smile: (which is actually very handy for interactive testing but not the best example :sweat_smile: )

### CODE REVIEW

File: .github/workflows/fledge.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install fledge
+        run: Rscript -e 'install.packages("fledge")'
+
+      - name: Bump version
+        run: Rscript -e 'fledge::bump_version(which = "dev")'
```
Comment (krlmlr) [inline .github/workflows/fledge.yaml:26]: Will this work? Should we:

- install from `"."`
- add a flag to `bump_version()` ?

Comment (maelle) [reply .github/workflows/fledge.yaml:26]: > install from `"."`

oh right fledge repo itself is a special case.

> add a flag to `bump_version()` ?

what do you mean?

Comment (krlmlr) [reply .github/workflows/fledge.yaml:26]: `fledge::bump_version(no_change_behavior = "noop")`

**Comment by krlmlr** [comment]
Thanks! Let's see how this flies.

**Pull request merged**